middleware.js

```diff
@@ -44,20 +46,23 @@ export async function middleware(req) {
         },
         body: JSON.stringify(body)
       });
+      time = (new Date().getTime()) - startTime;
 
     } else {
+      const startTime = new Date().getTime();
       response = await fetch(req.url, {
         headers: {
           "Cookie": cookie
         }
       });
+      time = (new Date().getTime()) - startTime;
     }
 
     const data = await response.json();
 
     // attempt to send metric
     // will timeout after 150ms
-    Promise.any([createTimeoutPromise(150), sendMetric(HOST_NAME, `${response.status}.${_metricName}`)])
+    Promise.any([createTimeoutPromise(150), sendMetric(HOST_NAME, `${response.status}.${_metricName}`, time)])
```

Comment:
I believe this should instead be a tuple of three promises, not two - one timeout, time for the timer, one for the counter